src/box/lua/schema.lua

```diff
@@ -1099,8 +1225,6 @@ function box.schema.space.bless(space)
         return tonumber(ret)
     end
     index_mt.__len = index_mt.len -- Lua 5.2 compatibility
-    index_mt.__newindex = function(table, index)
```

Comment:
Add a test for it, please.

Reply:
Testing what? Adding a new property to `index_mt` and `space_mt`?

Reply:
Yes. That it does not raise error.